### PR TITLE
Update cnos_backup.py

### PR DESCRIPTION
**Pull request by amuraleedhar**
##### SUMMARY
In this PR I am moving the code which is in util file cnos.py to respective module code. There was a review comment is this regard that util file is of huge size. This is an effort to reduce the size.

ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
lib\ansible\modules\network\cnos\cnos_backup.py

##### ANSIBLE VERSION
ansible 2.7.0.dev0 (devel f9cbdcd) last updated 2018/07/03 14:55:43 (GMT +550)
config file = /etc/ansible/ansible.cfg
configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
ansible python module location = /home/ansible/sheru/ansible/lib/ansible
executable location = /home/ansible/sheru/ansible/bin/ansible
python version = 2.7.6 (default, Nov 23 2017, 15:49:48) [GCC 4.8.4]

##### ADDITIONAL INFORMATION
Unused code in cnos.py will be removed later together with few more modules

### CODE REVIEW

File: lib/ansible/modules/network/cnos/cnos_backup.py [hunk 1]
```diff
@@ -33,21 +33,23 @@
...
-short_description: Backup the current running or startup configuration to a remote server on devices running Lenovo CNOS
+short_description: Backup the current running or startup configuration to a
+ remote server on devices running Lenovo CNOS
 description:
     - This module allows you to work with switch configurations. It provides a
      way to back up the running or startup configurations of a switch to a
      remote server. This is achieved by periodically saving a copy of the
      startup or running configuration of the network device to a remote server
-     using FTP, SFTP, TFTP, or SCP. The first step is to create a directory from
+     using FTP, SFTP, TFTP, or SCP. The first step is to create directory from
```
Comment (Qalthos) [inline lib/ansible/modules/network/cnos/cnos_backup.py]: Did you mean to drop the word 'a' here?

**Review by Qalthos — CHANGES_REQUESTED** [review]
> Unused code in cnos.py will be removed later together with few more modules

I think I get _why_ you are doing this, if you've been making edits to module_utils/cnos and it is hard to filter out what changes go for which module PR, but it makes this much harder to review and will make the later PR with many unrelated changes to the module_utils code _much_ harder to review. I feel it should not be too onerous to ask you to keep related changes together.

File: lib/ansible/modules/network/cnos/cnos_backup.py (hunk 1 above)
Comment (amuraleedhar) [reply lib/ansible/modules/network/cnos/cnos_backup.py]: Will add the 'a' back

Comment (amuraleedhar) [reply lib/ansible/modules/network/cnos/cnos_backup.py]: Will also remove the unwanted code from cnos.py along with this PR

**Comment by amuraleedhar** [comment]
Done as per the review coments

**Review by Qalthos — APPROVED** [review]

**Pull request merged**